packages/zwave-js/src/lib/driver/Driver.ts

```diff
@@ -2047,6 +2044,8 @@ ${handlers.length} left`,
 					if (!node.keepAwake) {
 						this.debounceSendNodeToSleep(node);
 					}
+					// The node must be awake because it answered
+					node.markAsAwake();
```

Comment:
This looks odd, the previous line would be sending the node to sleep, ergo it would not be awake anymore... right? should this be markAsAlive()?

Reply:
Not entirely correct. The previous line will actively send the node to sleep after 1 second if there are no messages for it in the queue. Until then, it is awake.

Reply:
right! because this is fast :)

Reply:
And depending on the queue length, the "send to sleep" might take pretty long until it actually happens.